### DISCUSSION TITLE
Freesurfer with BrainVISA 5.0.2

### DISCUSSION THREAD

**Discussion by f4bry** [brainvisa/brainvisa.github.io#68] [Q&A] [answered]
Hi,
I am trying to use the Freesurfer toolbox with BrainVISA 5.0.2.
Freesurfer home (`/usr/local/freesurfer`) is in the PATH but can't be seen by BrainVISA.

I am on Ubuntu 20.04 and I installed BrainVISA using singularity 3.8.0
Any suggestion?
Thanks,
Fab

**Comment by denisri** [comment, marked as answer]
Yes, BrainVisa distribution is now containerized, that means that `/usr/local` is the one from the containerized system, not your "host" system.
You can access the host system in `/host`, thus use `/host/usr/local/freesurfer` instead of `/usr/local/freesurfer`. You have to set it in the **container home config** (`~/.bashrc` in the container). But don't add all of `/host/usr/local/bin` in the `PATH` of the container: it would make the container use things installed for the host system, which may be incompatible.

**Comment by sapetnioc** [comment]
BrainVISA runs in a container. One could see a container as another computer with its own file system but with the possibility to have access to the other file system (your file system with `/usr/local/freesurfer`). But this link between the two file systems must be configured. Hopefully, we already expose the root of the host file system in the `/host` directory in the container. Therefore, you should see Freesurfer in `/host/usr/local/freesurfer`. But you can also make a dedicated link to make it appear with the same path (`/usr/local/freesurfer`) in the container. You can add directories mapping graphically using the `bv` command without parameters.

**Comment by f4bry** [comment]
Makes sense, thanks a lot!
Fab
